vesc_driver/src/vesc_driver.cpp

```diff
@@ -171,14 +179,39 @@ void VescDriver::vescPacketCallback(const std::shared_ptr<VescPacket const> & pa
     state_msg.state.distance_traveled = values->tachometer_abs();
     state_msg.state.fault_code = values->fault_code();
 
+    state_msg.state.pid_pos_now = values->pid_pos_now();
+    state_msg.state.controller_id = values->controller_id();
+
+    state_msg.state.ntc_temp_mos1 = values->temp_mos1();
+    state_msg.state.ntc_temp_mos2 = values->temp_mos2();
+    state_msg.state.ntc_temp_mos3 = values->temp_mos3();
+    state_msg.state.avg_vd = values->avg_vd();
+    state_msg.state.avg_vq = values->avg_vq();
+
     state_pub_->publish(state_msg);
   } else if (packet->name() == "FWVersion") {
     std::shared_ptr<VescPacketFWVersion const> fw_version =
       std::dynamic_pointer_cast<VescPacketFWVersion const>(packet);
     // todo: might need lock here
     fw_version_major_ = fw_version->fwMajor();
     fw_version_minor_ = fw_version->fwMinor();
+
+    RCLCPP_INFO(
+      get_logger(),
+      "-=%s=- hardware paired %d",
+      fw_version->hwname().c_str(),
+      fw_version->paired()
+    );
   }
+
+  auto & clk = *this->get_clock();
+  RCLCPP_INFO_THROTTLE(
+    get_logger(),
+    clk,
+    5000,
+    "%s packet received",
+    packet->name().c_str()
```

Comment:
This is definitely a debugging message and not something that needs to be `_INFO`. Please change to `_DEBUG` or remove.

Reply:
same from 654ba11. Could be removed.